lib/exercises.rb

```diff
@@ -1,19 +1,59 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n2)
+# Space Complexity: O(n)
 
 def grouped_anagrams(strings)
```

Comment:
👍 This works, but you can do better by sorting the words and using the sorted letters as keys for all the anagrams